.gitignore

```diff
@@ -22,3 +22,6 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+
+dist
+.clasp.json
```

Comment:
I never intended to initiate an Apps Script project with clasp from within the directory. The approach would be to configure the environment files and then run `npm run env:<env_name>`. If you would like to add initiating a project with clasp, we would need a different approach without overriding the workflow of the environemnts.